core/templates/dev/head/filters/string-utility-filters/normalize-whitespace.pipe.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview NormalizeWhitespace pipe for Oppia.
+ */
+
+import { UtilsService } from 'services/UtilsService.ts';
```

Comment:
Remove `.ts` and import the angular files first.

Reply:
Done